src/js/formats/helpers/buildCarousel.js

```diff
@@ -1,100 +1,73 @@
 import q from '../../lib/query';
+import buildSlider from './slider';
 
-export default function buildCarousel(slideCount) {
-    let currentSlide = 0;
+export default function buildCarousel(length) {
+    const slider = buildSlider({
+        length,
+    });
 
-    function clearCurrent() {
-        q('.js-current').forEach((el) => {
-            el.classList.remove('js-current');
-            el.classList.remove('explainer__carousel-slide--current');
-        });
+    function updateIndicator(currentSlide) {
         q('.js-active').forEach((el) => {
             el.classList.remove('explainer__carousel-indicator--active');
             el.classList.remove('js-active');
             el.classList.add('explainer__carousel-indicator--passive');
         });
-    }
 
-    function setCurrent() {
-        q(`#carousel-slide-${currentSlide}`).forEach((el) => {
-            el.classList.add('js-current');
-            el.classList.add('explainer__carousel-slide--current');
-        });
         q(`#carousel-indicator-${currentSlide}`).forEach((el) => {
```

Comment:
what do you reckon about ditching the id altogether and using a `[data-` selector? we generally avoid ids as far as we can